### PR TITLE
Module 5 Project Exercise

### PR DESCRIPTION
**Pull request by pat-morris**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,21 +1,43 @@
 # DevOps Apprenticeship: Project Exercise
```
Comment (Jonesey13) [inline README.md:1]: There is no documentation on how to the run the docker containers here (this is a minimum requirement for this to pass review)

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+version: "3.8"
+
+services:
+  todo-app:
```
Comment (Jonesey13) [inline docker-compose.yml]: There is no bind mount to the project folder. This prevents hot-reloading from working and prevents this from working as a fully effective development container. This is also a blocker for this passing review. Please see page 6 from the exercise brief for some clues on how to set this up.

File: pyproject.toml [hunk 3]
```diff
@@ -9,11 +9,16 @@ python = "^3.8"
...
 
 [tool.poetry.dev-dependencies]
+pytest = "^6.0.2"
+selenium = "^3.141.0"
 
 [build-system]
 requires = ["poetry>=0.12"]
 build-backend = "poetry.masonry.api"
+
+[virtualenvs]
```
Comment (Jonesey13) [inline pyproject.toml:22]: I believe this has no effect unless it's in the `poetry.toml` file.

**Review by Jonesey13 — CHANGES_REQUESTED** [review]
You're most of the way there. Just need to get hot reloading working and document how to run the containers in the `README.md` for this to pass.

File: README.md (hunk 1 above)
Comment (pat-morris) [reply README.md:1]: Instructions added to run docker containers

File: applogs.txt [hunk 4]
```diff
@@ -4,5 +4,9 @@
  * Running on http://0.0.0.0:5000/ (Press CTRL+C to quit)
```
Comment (hugh-emerson) [inline applogs.txt:4]: This file shouldn't be committed.

**Review by hugh-emerson — APPROVED** [review]
Looks good. Merge when you're ready.

**Pull request merged**